src/frontend/src/lib/utils/nav.utils.ts

```diff
@@ -24,17 +19,15 @@ export const isSubRoute = (page: Page): boolean =>
 	isRouteTransactions(page) || isRouteSettings(page);
 
 const tokenUrl = ({
-	token: { name },
-	networkId,
+	token: { name, network },
```

Comment:
If interested, here you can destruct the `network` that way the content of the function does not have to change.

```
const tokenUrl = ({
   token: { name, network: {id: networkId} },
```

Reply:
even better! changing it